src/components/TopNav.tsx

```diff
@@ -1,20 +1,322 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import AppBar from "@mui/material/AppBar";
```

Comment:
I suggest keeping imports for MUI components separate from other imports and commented with:
// Import MUI components

Also, listing them in alphabetical order makes it easier to find what you need quickly and looks more neat and professional